src/shared/utils/index.js

```diff
@@ -105,18 +124,20 @@ const setChartMargin = (
     left
   ] = useAxis
     ? [
-      TEXT_HEIGHT / 2 + 1,
-      TEXT_HEIGHT / 2 + 1,
-      AXIS_TICK_WIDTH + BUFFER,
+      SYMBOL_SIZE / 2 + 1,
+      SYMBOL_SIZE / 2 + 1,
+      AXIS_TICK_WIDTH,
       AXIS_TICK_WIDTH
     ]
     : [
-      2 * BUFFER,
-      2 * BUFFER,
-      2 * BUFFER,
-      2 * BUFFER
+      BUFFER,
+      BUFFER,
+      BUFFER,
+      BUFFER
```

Comment:
```js
new Array(4).fill(BUFFER)
```

this would be a bit less error prone, where the number of items is more apparent